skycoin/skywire
# Release v0.5.0

Did you run `make format && make check`?


 Changes:	
- 

How to test this PR:
1. Run `./skywire-visor skywire-config.json`
2. Check if the config is updated to 1.1.0 and the URLs are updated.
3. Check if there is no other error
